cgi-bin/DW/Controller/Manage/Profile.pm

```diff
@@ -0,0 +1,460 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Manage::Profile
+#
+# This code is based on code originally created by the LiveJournal project
+# owned and operated by Live Journal, Inc. The code has been modified and
+# expanded by Dreamwidth Studios, LLC. These files were originally licensed
+# under the terms of the license supplied by Live Journal, Inc, which made
+# its code repository private in 2014. That license is archived here:
+#
+# https://github.com/apparentlymart/livejournal/blob/master/LICENSE-LiveJournal.txt
+#
+# In accordance with the original license, this code and all its
+# modifications are provided under the GNU General Public License.
+# A copy of that license can be found in the LICENSE file included as
+# part of this distribution.
+#
+# Authors:
+#      Jen Griffin <kareila@livejournal.com>
```

Comment:
really? :)

Reply:
At least I remembered to change the package name in the header this time?